### PR TITLE
Scrape decisions from the Trial Decision Library.

### PR DESCRIPTION
**Pull request by emspishak**
This scrapes the table at https://nypdonline.org/link/1016 . Most of these should already be included in the profile data, but in the case a trial decision gets posted when an officer no longer works for the department (e.g. they got fired) it'll show up here but not in the profile data (because their profile goes away when they leave).

### CODE REVIEW

**Comment by ryanwatkins** [comment]
Thanks for the addition the functionality looks good. I only have a couple trivial comments on style preferences.

File: script.js [hunk 1]
```diff
@@ -513,6 +514,60 @@ async function saveProfiles({ letter, officers }) {
...
+// profile data (because their profile goes away when they leave).
+async function scrapeTrialDecisions() {
+  headers.Cookie = await getCookie()
+
+  const result = await scheduleFetch({
+    url: reportList.trialDecisions,
+    options: { method: 'GET', headers },
+  })
+
+  const data = []
```
[the remark this answers is not in the document]
Comment (emspishak) [reply script.js]: Done. Makes sense!

File: script.js [hunk 2]
```diff
@@ -513,6 +514,60 @@ async function saveProfiles({ letter, officers }) {
...
+      map: {
+        names: 'aa970ef7-62f1-4d89-bcb8-5078017ee41a',
+        date: '8a6d6957-716f-4f3c-998f-e4aebbdbe912',
+        url: '0be5eacf-3870-4351-aa42-439067baadbe',
+      },
+    })
+    doc.url = 'https://oip.nypdonline.org' + doc.url.match(/<a href="([^"]+)"/)[1]
+
+    const nameCol = row.Columns.find(item => item.Id === 'aa970ef7-62f1-4d89-bcb8-5078017ee41a')
+    var taxids = []
```
[the remark this answers is not in the document]
Comment (emspishak) [reply script.js]: Done.

File: script.js [hunk 3]
```diff
@@ -513,6 +514,60 @@ async function saveProfiles({ letter, officers }) {
...
+    if (nameCol.ColumnValue) {
+      taxids = nameCol.ColumnValue.split(',')
+    }
+    var taxidIndex = 0;
```
[the remark this answers is not in the document]
Comment (emspishak) [reply script.js]: Done

File: script.js [hunk 4]
```diff
@@ -513,6 +514,60 @@ async function saveProfiles({ letter, officers }) {
...
+    doc.officers = doc.names.split('; ').map(name => {
+      const [last_name, first_name] = name.replace(/<u>/g, '').replace(/<\/u>/g, '').split(', ')
+      const officer = {
```
[the remark this answers is not in the document]
Comment (emspishak) [reply script.js]: Done

File: script.js [hunk 5]
```diff
@@ -513,6 +514,60 @@ async function saveProfiles({ letter, officers }) {
...
+  headers.Cookie = await getCookie()
+
+  const result = await scheduleFetch({
+    url: reportList.trialDecisions,
+    options: { method: 'GET', headers },
+  })
+
+  const data = []
+  for (const row of result) {
+    const doc = findValues({
```
[the remark this answers is not in the document]
Comment (emspishak) [reply script.js]: Done

**Pull request merged**